homeassistant/components/samsungtv/media_player.py

```diff
@@ -62,7 +61,7 @@
     {
         vol.Required(CONF_HOST): cv.string,
         vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
-        vol.Optional(CONF_PORT, default=DEFAULT_PORT): cv.port,
+        vol.Optional(CONF_PORT): cv.port,
```

Comment:
Is removing the default port a breaking change? What happens now if users don't specify port?

Reply:
That's when we try to find method automatically, right? Should we mention this in the breaking change paragraph? To make it clear to users in the release notes.

Reply:
Ok.